requirements/default.txt

```diff
@@ -315,6 +318,14 @@ idna==2.10 \
     --hash=sha256:b307872f855b18632ce0c21c5e45be78c0ea7ae4c15c828c20788b26921eb3f6 \
     --hash=sha256:b97d804b1e9b523befed77c48dacec60e6dcb0b5391d57af6a65a312a90648c0 \
     # via -r default.in, requests
+importlib-metadata==1.7.0 \
```

Comment:
Importlib-metadata, importlib-resources and zipp are not added for me by just adding django-pipeline in default.in